JokeRandomizer/JokeRandomizer/ViewModel/JokesViewModel.swift

```diff
@@ -7,46 +7,57 @@
 
 
 import Foundation
+import Combine
 
-class JokesViewModel: ObservableObject {
+class JokesViewModel {
+    private var cancellables = Set<AnyCancellable>()
+    var refreshButtonStyle, favoriteButtonStyle, listButtonStyle: CustomButton.ButtonTypes
```

Comment:
can be let instead of var since I don't see this being set other than inside init